package.json

```diff
@@ -51,14 +53,16 @@
     "typescript": "^5.2.2"
   },
   "devDependencies": {
-    "@types/vscode": "^1.84.0",
+    "@swc/cli": "^0.1.63",
+    "@swc/core": "^1.3.96",
     "@types/mocha": "^10.0.3",
     "@types/node": "18.x",
+    "@types/vscode": "^1.84.0",

```

Comment:
I suppose this version will be downgraded to 1.80 due to supporting  compatibility.
This pr will get a conflict after merging #22 .

Reply:
I fixed conflict on package.json directly.
Check the last commit i made on this PR.